src/resources/i18n/translations/en.json

```
@@ -17,16 +17,16 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount may be higher due to network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add custom RSS feed",
-  "Add custom RSS feeds": "Add custom RSS feeds",
+  "Add current item next in queue when playing a new item": "Start playback immediately before continuing",
```

```suggestion
  "Add current item next in queue when playing a new item": "Play right away, continue queue afterwards.",
```
I think this is the one.